bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -134,69 +134,48 @@ <h3 class="c-sticky-cta-title-sub">{{ _('Live your life. Own your life.') }}</h3
-          aspect_ratio='mzp-has-aspect-1-1',
-          link_url=url('firefox.concerts')
+      {{ card(
+        tag_label=_('Internet Health'),
+        title=_('Goodbye, EdgeHTML'),
+        ga_title='Goodbye EdgeHTML',
+        desc=_('Microsoft is giving up on an independent shared platform for the internet, and in doing so, hands over control of more of online life to Google.'),
+        image_url='home/2018/cards/edge.png',
+        include_highres_image=True,
+        aspect_ratio='mzp-has-aspect-1-1',
+        link_url='https://blog.mozilla.org/blog/2018/12/06/goodbye-edge/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card'
         )}}
-      {% else %}
-        {{ card(
-          tag_label=_('News'),
-          title=_('Firefox on The Cheddar'),
-          ga_title='Firefox on The Cheddar',
-          desc=_('We want to be there to help protect you in your web experience — Denelle Dixon on why we built enhanced tracking protection into Firefox.'),
-          image_url='home/2018/cards/cheddar.jpg',
-          aspect_ratio='mzp-has-aspect-1-1',
-          link_url='https://blog.mozilla.org/internetcitizen/2018/11/05/cheddar-firefox-enhanced-tracking-protection/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card'
-          )}}
-      {% endif %}
         <!-- 3 -->
         {{ card(
-          tag_label=_('Podcast'),
-          title=_('IRL returns with Manoush Zomorodi'),
-          ga_title='IRL returns with Manoush Zomorodi',
-          desc=_('Explore the potential of ethical tech and how people everywhere can take the wheel back to enjoy a more citizen-centric web.'),
-          image_url='home/2018/cards/nov/irl.jpg',
+          tag_label=_('Firefox'),
+          title=_('Get the latest Firefox'),
+          ga_title='Get the latest Firefox',
+          desc=_('This version of Firefox for desktop has a couple new features that simplify and personalize your online experience.'),
+          image_url='home/2018/cards/dec/firefox.jpg',
           include_highres_image=True,
           aspect_ratio='mzp-has-aspect-16-9',
-          link_url='https://blog.mozilla.org/internetcitizen/2018/11/07/irl-podcast-manoush-zomorodi/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card',
+          link_url='https://blog.mozilla.org/blog/2018/12/11/latest-firefox-release-available-today/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card',
         )}}
         <!-- 4 -->
         {{ card(
           tag_label=_('Privacy'),
           title=_('Better watch out'),
           ga_title='Better watch out',
-          desc=_('On a scale from 1 to 10, how creepy are the gifts on your wish list this year?'),
-          image_url='home/2018/cards/nov/privacy.jpg',
+          desc=_('On a scale from 😄 to 😧, how creepy are the gifts on your wish list this year?'),
```

Comment:
I'm a little unsure how older browsers may display emoji? I'm guessing this is relatively safe but we've never really tested this out. Not a blocker, just noting for reference.

Reply:
Probably little squares for IE8 but they must be used to that by now.